src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -85,6 +111,16 @@ public static class PropertyNames
             ///     default: "" (no authentication).
             /// </summary>
             public const string SchemaRegistryBasicAuthUserInfo = "schema.registry.basic.auth.user.info";
+
+            /// <summary>
+            ///     Key subject name strategy.
+            /// </summary>
+            public const string SchemaRegistryKeySubjectNameStrategy = "schema.registry.key.subject.name.strategy";
```

Comment:
The `SchemaRegistry` prefix on property names here, and in the config class itself are pretty annoying. I'm thinking we should depreciate them and add variants without this prefix (however the naming of the underlying property name `schema.registry.key.subject.name.strategy` should remain). There's plenty of color to add to all this, available verbally to any reviewer on request...

Reply:
We need something to separate the client and SR config namespaces, right?

Reply:
Only for configuration specified in a flat configuration file, which keeping the `schema.registry.key.subject.name.strategy` names enables. This is being overly accommodating though because no one's going to do that in practice in C#. Having `SchemaRegistry` in the property names as well as the enum (which always needs to be specified) is just unnecessarily doubling up.

I'll possibly open a follow up PR with the above change.